cli/src/commands/parse.rs

```diff
@@ -181,7 +181,26 @@ fn find_manifest_format(path: &Path) -> Option<(LockfileFormat, Option<PathBuf>)
 
     // Look for formats which already have a lockfile generated.
     let manifest_lockfile = formats.find_map(|format| {
+        // Check the root directory for a lockfile
+        let root_dir_entries = match fs::read_dir(manifest_dir) {
+            Ok(entries) => entries,
+            Err(_) => return None,
+        };
+
+        for entry_result in root_dir_entries {
+            match entry_result {
+                Ok(entry) => {
+                    if format.parser().is_path_lockfile(&entry.path()) {
+                        return Some((format, entry.path()));
+                    }
+                },
+                Err(_) => continue,
+            }
+        }
```

Comment:
This is not a fix to the problem. Now you have the same issue just one level down. There's no guarantee that the lockfile will actually be at the root level.

Reply:
Okay. I reopened the issue.